src/main/kotlin/io/appwrite/Client.kt

```diff
@@ -47,7 +47,7 @@ class Client @JvmOverloads constructor(
     init {
         headers = mutableMapOf(
             "content-type" to "application/json",
-            "x-sdk-version" to "appwrite:kotlin:0.0.0",            
+            "x-sdk-version" to "appwrite:kotlin:0.0.1",            
             "x-appwrite-response-format" to "0.8.0"
```

Comment:
Not sure but this might need to be changed to 0.9.0 based on the readme updates

```suggestion
            "x-appwrite-response-format" to "0.9.0"
```

Reply:
Done :+1:  :)